wiki/Manual.md

```diff
@@ -370,6 +370,14 @@ console.log(exchange.features);
 }
 ```
 
+There is also an old approach by checking `.has` property, but it's not as precise as `.features`, eg: 
```

Comment:
i used here `old` just for the chronological meaning as `.features` is newer functionality and `.has` has been there for a long time,